### DISCUSSION TITLE
add read_only_all/write_only_all to `struct ObjectArgs` in `#[derive(Object)]`

### DISCUSSION THREAD

**Discussion by pymongo** [poem-web/poem#70] [Ideas]
current:

```rust
#[derive(poem_openapi::Object)]
pub struct LoginForm {
    #[oai(write_only)]
    pub username: String,
    #[oai(write_only)]
    pub password: String,
    // ... twenty fields ...
}
```

if my LoginForm has multi fields, e.g. has 20 fields, I need to write `#[oai(write_only)]` to each field twenty times

the simpify I want(auto add write_only to each field):

```rust
#[derive(poem_openapi::Object)]
#[oai(write_only_all)]
pub struct LoginForm {
    pub username: String,
    pub password: String,
    // ... twenty fields ...
}
```

**Comment by pymongo** [comment]
My thought of write_only_all rule:

field.write_only = ObjectArgs.write_only_all || ObjectField.write_only

field.read_only = ObjectArgs.read_only_all || ObjectField.read_only

**Reply by sunli829** [reply]
It looks good, would you like to open a PR for this? 😁
